libcontainer/cgroups/cgroups.go

```diff
@@ -1,9 +1,24 @@
 package cgroups
 
 import (
+	"errors"
+
 	"github.com/opencontainers/runc/libcontainer/configs"
 )
 
+var (
+	// ErrNoDevRules is an error returned when a cgroup manager
+	// is not configured to set device rules.
+	ErrNoDevRules = errors.New("cgroup manager is not configured to set device rules")
```

Comment:
`ErrDevicesUnsupported` would probably be a clearer name for the error. `ErrNoDevRules` seems to be indicating the opposite of what the issue is (there are devices rules but no devices manager configured).

Reply:
Right; renamed.